sopel/modules/url.py

```diff
@@ -63,6 +63,8 @@ def configure(config):
     | exclude | https?://git\\\\.io/.* | A list of regular expressions for URLs for which the title should not be shown. |
     | exclusion\\_char | ! | A character (or string) which, when immediately preceding a URL, will stop the URL's title from being shown. |
     | shorten\\_url\\_length | 72 | If greater than 0, the title fetcher will include a TinyURL version of links longer than this many characters. |
+    | enable\\_private\\_resolution | True | Enable DNS resolution for all domains to validate if there are RFC1918 resolutions. |
+    | enable\\_dns\\_resolution | True | Enable DNS resolution for all domains to validate if there are RFC1918 resolutions. |
```

Comment:
By convention, this example usually includes the default value of each setting. (There are existing exceptions in this module.) Both of these new settings should be `False`, by that convention.